src/server.js

```diff
@@ -1,7 +1,11 @@
 const http = require('http');
 const router = require('./router');
-const port = 3000;
+
 const server = http.createServer(router);
+const hostname = process.env.HOSTNAME || 'localhost';
+const port = process.env.PORT || 3000;
+
 
-server.listen(port, ()=> console.log(`this server runs on port ${port}`)
-);
+server.listen(3000, () => {
```

Comment:
the point of defining port is to use it down in the server listener. We should replace 3000 with port.